tests/conftest.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
```

Comment:
Just to document some of the ideas that we discussed, we should look into segregation of test types by mapping them directly to the directory they live in. By following the composite structure of the repo, unit tests would strictly live in tests/resources, the next level of nightly/smoke testing would live in the task layer. Functionality and tests would increase in scope, duration and complexity as we add more layers of abstraction.

Reply:
I've slept over it and I'm still not quite sure about this approach for the following reason: except for project maintainers no one will guess this intrinsic relation. To be honest, I hadn't realized it until you told me :/. I'd like to hear your opinions here, @Dannyb48 , @amacmurr .

Reply:
I do like the idea of having unit tests, that do not rely on having an active endpoint available. However, I believe that we need to have further discussions surrounding that approach before we dive into it. There have also been discussions regarding further aligning the directory structure to the product. I think we need to sort things out a bit before proceeding further.